Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy.swift

```diff
@@ -170,7 +148,7 @@ struct PathHierarchy {
                 }
             }
 
-            for relationship in relationships where relationship.kind == .overloadOf {
+            for relationship in graph.relationships where relationship.kind == .overloadOf {
```

Comment:
If I understand correctly we should be able to remove this entire for loop since the individual symbol graph files wouldn't contain any `.overloadOf` relationships anymore.

Reply:
Similarly, I believe that we should be able to remove [this if-statement](https://github.com/swiftlang/swift-docc/pull/986/files/3a3cfa9ef8090e932e400a36fa615fdbd47d0443..68f80903e8c8ab72230eab57326a4ccac87c5a91#diff-e0d0f88acc1bca134d47867a977e60a988a20118c4b52451f90376f6e3d56f7cR123-R131) as well since the individual symbol graph files wouldn't contains any overload group symbols—with `overloadGroupIdentifierSuffix` suffixes in their unique identifiers—anymore.

Reply:
Ah yeah, good catch. I'll take those out.